eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -41,7 +43,10 @@ param(
   [string]$Language,
 
   [Parameter(Mandatory = $true)]
-  [string]$RepoId
+  [string]$RepoId,
+
+  [Parameter(Mandatory = $false)]
+  [string]$VsoVariable
```

Comment:
If a user were to run this locally they would need to modify environment variables. Instead this should just be the JSON string. 

The name should also be something like `UserInfoJson` and the documentation should show an example JSON object with fake information: 

```
.PARAMETER UserInfoJson
User information JSON of the form: 
{"GithubUserName":"githubuser","AadId":"<uuid>","Name":"Full Name","Alias":"useralias","AadUpn":"useralias@contoso.com"}
```

Invocation could look like: 

```yaml
...
- pwsh: |
    $packageInfoJson = '${{ convertToJson(parameters.PackageInfoLocations) }}'.Trim('"').Replace("\\", "/")
    $packageInfoLocations = ConvertFrom-Json $packageInfoJson
    ${{ parameters.ScriptDirectory }}/Update-DocsMsMetadata.ps1 `
      -PackageInfoJsonLocations $packageInfoLocations `
      -DocRepoLocation "$(DocRepoLocation)" `
      -Language '${{parameters.Language}}' `
      -RepoId '${{ parameters.RepoId }}'
      -UserInfoJson '$(UserInfoJsonFromIdentityResolver)'      # Instead of VsoVariable
  displayName: Apply Documentation Updates
...
```